collector/src/main/java/com/bitmovin/analytics/data/AdEventData.kt

```diff
@@ -96,6 +96,7 @@ data class AdEventData(
         var playerTech: String? = null,
         var screenHeight: Int? = null,
         var screenWidth: Int? = null,
+        var screenOrientation: String? = null, // SVARGA: find how it is serialized and sent
```

Comment:
What's the `// SVARGA` ?

Reply:
my custom TODO that should be deleted there 😅